src/app/components/CourseDetailHeader/CourseDetailHeader.module.css

```diff
@@ -0,0 +1,29 @@
+.header {
+  display: grid;
+  grid-template-columns: auto auto;
+  column-gap: 0.5em;
+  margin: 0;
+  padding: 0;
+  width: 80%;
+}
+
+.header__image {
+  grid-column: 1/2;
+  grid-row: 1/4;
+}
+
+.header__text {
+  display: grid;
+  grid-template-rows: auto auto auto;
+  grid-column: 2/3;
+}
+
+.header__text__body {
```

Comment:
This (andother classes) is not BEM like:
https://en.bem.info/methodology/faq/#why-not-create-elements-of-elements-block__elem1__elem2

Reply:
Helpful! Thanks!